src/main/java/io/vertx/ext/jdbc/impl/JDBCConnectionImpl.java

```diff
@@ -128,4 +129,57 @@ public SQLConnection rollback(Handler<AsyncResult<Void>> handler) {
     new JDBCRollback(vertx, conn, context).execute(handler);
     return this;
   }
+
+  @Override
+  public SQLConnection setTransactionIsolation(TransactionIsolation isolation) {
+    try {
+      switch (isolation) {
+        case READ_COMMITTED:
+          conn.setTransactionIsolation(Connection.TRANSACTION_READ_COMMITTED);
```

Comment:
Thanks for this!

Just one question.
I suppose setTransactionIsolation is very similar with setAutoCommit, both of them need to talk with the actual server. So this call could be blocked by network operation.
setAutoCommit use executeBlocking to run the underlying code, my question is why this one doesn't need executeBlocking? Am I missing something here?

Thanks!